void_migration/void_migration.py

```diff
@@ -30,6 +30,9 @@ def time_march(p):
 
     plotter.set_plot_size(p)
 
+    s_ms = [0, 0]
```

Comment:
i thought we were going to plot the concentration fields directly and stop using this hack?

Reply:
done